src/market/directory.rs

```diff
@@ -6,15 +6,15 @@
 
 /// Represents the Tor address and port configuration.
 // It should be set to your specific Tor address and port.
-pub const TOR_ADDR: &str = "127.0.0.1:9050";
+pub const TOR_SOCKS_ADDR: &str = "127.0.0.1:19050";
```

Comment:
Tor socket port needs to be a configuration parameter. The default value could be the usual 19050/9050. But it should be a configuration option for user to set.